itests/net/connection.go

```diff
@@ -0,0 +1,53 @@
+package net
+
+import (
+	"bufio"
+	"fmt"
+	"github.com/wavesplatform/gowaves/pkg/proto"
+	"net"
+	"time"
+)
+
+type OutgoingPeer struct {
+	conn net.Conn
+}
```

Comment:
I can't find `Close` method for this struct. Do you close connections after using?